src/data/basic_dataset.py

```diff
@@ -75,10 +71,13 @@ def __init__(
                         image_paths["CAM_FRONT"],
                     )
 
-                # NOTE: This is a simple workaround if we do not have all files available
-                if not os.path.isfile(image_path):
-                    removed += 1
-                    continue
+                with Image.open(image_path) as image:
+                    image_width, image_height = image.size
+                    if resize_factor != 1.0:
+                        image_width = int(image_width * resize_factor)
+                        image_height = int(image_height * resize_factor)
+
+                    max_pixels = image_width * image_height
```

Comment:
This is not how max pixels is calculated. Have a look at the docs or how it is set in the QwenVL engine in our code. The naming for this variable is shit, but what they mean by image pixels is the number of visual tokens in an image, not the amount of actual pixels.